standalone/phost/src/main.rs

```diff
@@ -582,7 +575,15 @@ async fn bridge(args: Args) -> Result<()> {
         if !info.initialized {
             warn!("pRuntime not initialized. Requesting init...");
             runtime_info = Some(
-                init_runtime(&client, &pr, !args.ra, args.use_dev_key, &args.inject_key).await?,
+                init_runtime(
+                    &client,
+                    &pr,
+                    !args.ra,
+                    args.use_dev_key,
+                    &args.inject_key,
+                    args.operator.clone(),
```

Comment:
Let's use [`from_str`](https://substrate.dev/rustdocs/v3.0.0-monthly-2021-07/src/sp_core/crypto.rs.html#823-837) to allow ss58 encoded address:

```rust
let parsed_operator = sp_core::crypto::AccountId32::from_str(&args.operator)?;
let operator_hex = hex::encode(&parsed_operator);
```